ambry-rest/src/main/java/com/github/ambry/rest/PublicAccessLogHandler.java

```diff
@@ -218,7 +249,9 @@ private void logSSLInfo(ChannelHandlerContext ctx) {
                 X500Principal principal = ((X509Certificate) certificate).getSubjectX500Principal();
                 Collection subjectAlternativeNames = ((X509Certificate) certificate).getSubjectAlternativeNames();
                 sslLogMessage.append(", [principal=").append(principal).append("]");
+                structuredLogMessage.put("principal", principal.toString());
```

Comment:
Do we know why we had separate `sslLogMessage` and `logMessage` earlier. Should we also separate out `structuredLogMessage` for ssl and non ssl cases.

Reply:
I'm not quite sure what the original reasoning here was. However, if I understand the original code correctly, it simply uses `sslLogMessage` to build up an SSL-specific log statement which is then appended to `logMessage` at the end of the method. So `sslLogMessage` is not really a log message by itself, it seems to exist for first-pass convenience.